.travis/utils.sh

```diff
@@ -0,0 +1,21 @@
+setup_elasticsearch() {
```

Comment:
@dimagi/team-commcare-hq fyi, you can now write tests for elasticsearch. travis runs 0.90.13. also when running elasticsearch in tests, all the indexes will be prefixed with `test_` so you can add and remove data without messing up your data (though be warned it runs on the same port so if you make a call to delete every index, it will delete every index). we've simple wrapped all index definitions in `es_index` which will auto prefix `test_` when UNIT_TESTING is true. see here: https://github.com/dimagi/commcare-hq/pull/9488/files#diff-3dfba8147b36e63e4e36372b2a2cd179R8